app/controllers/users_controller.rb

```diff
@@ -3,31 +3,14 @@ class UsersController < ApplicationController
 
   def show
     @user = User.find(params[:id])
-  end
+    @dinosaurs = Dinosaur.where(user: @user)
 
-  def edit
-    @user = User.find(params[:id])
     if current_user != @user
       flash[:errors] = "You're not signed in as this user"
       redirect_to dinosaurs_path
     end
   end
 
-  def update
-    @user = User.find(params[:id])
-    if current_user != @user
-      flash[:errors] = "You're not signed in as this user"
-      redirect_to dinosaurs_path
```

Comment:
So this branch is a little behind some of the other things we have updated.  Mike let me know that we actually removed this piece in a later branch.  I think we should be good to go now because there is only one place that that logic is used.  Let me know if you still want me to extract a method.